Harvest.Web/ClientApp/src/Requests/AccountsInput.tsx

```diff
@@ -10,6 +10,7 @@ import { ProjectAccount } from "../types";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faMinusCircle } from "@fortawesome/free-solid-svg-icons";
 import { useIsMounted } from "../Shared/UseIsMounted";
+import { array } from "yup";
```

Comment:
Don't see where this is being used.

Reply:
It got added when I typed in array on accident. I will get rid of it